### DISCUSSION TITLE
repodaki dosya hakkında

### DISCUSSION THREAD

**Discussion by dabirtekk** [gusanmaz/BMB212-Algorithms-2223#12] [General]
davet linkinden oluşturduğum repoda student.java dosyasına erişemiyorum nereden bulabilirim?

**Comment by guvenrdvan** [comment]
Eğer yoksa senin oluşturman gerekiyor diye  anladım aşağıdaki repoların bir tanesinde

**Comment by gusanmaz** [comment]
Buraya bakabilirsin yanit icin: #2
